src/main.py

```diff
@@ -16,26 +15,21 @@
 
 
 def get_player_cookie(driver: uc_chrome) -> str:
-    sleep_count = 0
     player_cookie = ""
-    while sleep_count <= 7:
-        for resp in driver.requests():
-            if resp.url == "https://fantasy-api.formula1.com/f1/2022/sessions?v=1":
-                player_cookie = resp.response.headers.get("Set-Cookie").split(";")[0]
-                print(player_cookie)
-                return player_cookie
-        sleep(5)
-        sleep_count += 1
-    print("Error session")
+    request = driver.wait_for_request('/f1/2022/sessions', 60)
```

Comment:
This is amazing!
But we also have to handle `TimeoutException` if the request searched doesn't occur within the specified timeout.